sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/query.sql

```diff
@@ -344,6 +344,138 @@ metrics_org_mozilla_ios_fennec AS (
   WHERE
     mozfun.norm.truncate_version(client_info.app_display_version, 'major') >= 28
 ),
+-- metrics for Focus Android release
+metrics_org_mozilla_focus AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    client_info.client_id,
+    normalized_country_code AS country,
+    'Focus' AS app_name,
```

Comment:
In `nondesktop_clients_last_seen` the app names for Glean Focus/Klar products contain a `Baseline` to distinguish them from core pings. So `Android Focus Baseline`, `iOS Focus Baseline` etc. I think we might want to be consistent here
https://github.com/mozilla/bigquery-etl/pull/3040

Reply:
@scholtzan @ANich although in #3040 it was called `Baseline` because it was coming from the `baseline` ping. Here we're looking at the `Metrics` ping. Maybe we should have gone by "Android Focus Glean" :)

Reply:
We could do the renaming of "Baseline" to "Glean" in the views. Or we backfill the tables with the new name

Reply:
Updated